### PR TITLE
Add powerline-gitstatus recipe.

### PR DESCRIPTION
**Pull request by lebedov**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/powerline-gitstatus_conda_recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        - python
+        - powerline-status
+
+test:
+    imports:
+        - powerline_gitstatus
+
+about:
+    home: https://github.com/jaspernbrouwer/powerline-gitstatus/
+    license: MIT
```
Comment (chrisburr) [inline recipes/powerline-gitstatus_conda_recipe/meta.yaml]:
```suggestion
    license: MIT
    license_family: MIT
    license_file: LICENSE
```

File: recipes/powerline-gitstatus_conda_recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    version: {{version}}
+
+source:
+    url: https://github.com/jaspernbrouwer/powerline-gitstatus/archive/v{{version}}.tar.gz
+    sha256: f198a6947dcd76cc380e308e10a56d0988b1b3d4c69517bf141f4daa483fe367
+
+build:
+    noarch: python
+    number: {{build}}
+    script: {{PYTHON}} setup.py install
```
Comment (chrisburr) [inline recipes/powerline-gitstatus_conda_recipe/meta.yaml]: Generally using `pip` is better than running the `setup.py` file directly:
http://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=pip#use-pip

**Comment by lebedov** [comment]
Why isn't the PYTHON Jinja variable being recognized? conda-build 3.17.8 doesn't appear to recognize it on my local system either.

**Comment by dbast** [comment]
Is the version und number valid jinja? Maybe that the source for that issue...

You can get some inspiration to do the first half of the recipe a bit differently from the meta.yaml generated by `conda-skeleton pypi --noarch-python powerline-gitstatus`.

**Comment by dbast** [comment]
Also I guess the requirements currently in the `build` section better belong to the `host` section, as nothing is really built here.

**Comment by lebedov** [comment]
@dbast Thanks for the tip - seems to build properly now.

File: recipes/powerline-gitstatus_conda_recipe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{% set build = 0 %}
+{% set name = "powerline-gitstatus" %}
+{% set version = "1.3.1" %}
+
+package:
+    name: "{{name|lower}}"
```
Comment (mariusvniekerk) [inline recipes/powerline-gitstatus_conda_recipe/meta.yaml]:
```suggestion
    name: "{{ name|lower }}"
```
spacing nit

Comment (mariusvniekerk) [inline recipes/powerline-gitstatus_conda_recipe/meta.yaml]:
```suggestion
    name: "{{ name|lower }}"
```
spacing nit

**Pull request merged**